app/src/main/res/values/strings.xml

```diff
@@ -116,11 +116,15 @@
     <string name="about">About</string>
     <string name="about_text">Programmed by <a href="http://www.digitalia.be/">Christophe Beyls</a>.
```

Comment:
How about `Programmed by <a href="http://www.digitalia.be/">Christophe Beyls</a> and FOSSASIA Community (<a href="https://github.com/fossasia/fossasia-companion-android/blob/master/README.md">check contributors</a>)` ?